ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -64,9 +64,21 @@ azure_connection::put_object(const std::string &file_name) const
     return 0;
 }
 
+// Delete an object in the bucket given the object name.
 int
-azure_connection::delete_object() const
+azure_connection::delete_object(const std::string &object_key) const
 {
+    std::string obj = _object_prefix + object_key;
+
+    auto object_client = _azure_client.GetBlobClient(obj);
+    auto delete_blob_response = object_client.DeleteIfExists();
+
+    // Returns false if obj doesn't exist.
+    if (!delete_blob_response.Value.Deleted) {
+        std::cerr << obj + " : No such Object." << std::endl;
```

Comment:
np: I would place the comment as " : No such object file exists in the bucket."

Reply:
done :)